src/default-permalink-content-renderer.js

```diff
@@ -6,9 +6,10 @@ var styles = require('css!./css/styles.css');
 var sdkStyles = require('css!streamhub-sdk/css/style.css');
 var packageAttribute = require('./package-attribute');
 
-var defaultPermalinkContentHandler = function (content) {
+var defaultPermalinkContentHandler = function (content, opts) {
+    opts = opts || {};
     //Get the view for the content
-    var cvf = permalinkViewFactory(),
```

Comment:
Still works for me, the presence of opts.baseFactory in permalinkViewFactory threw me off, thinking the intention was to always apply certain behavior to views used with this permalink app.  Branch updated.